build/excludefiles.txt

```diff
@@ -6,9 +6,21 @@
 .travis.yml
 *.git*
 *.idea*
+app/console
```

Comment:
Delete of command above is solution. This line is not needed

Reply:
I agree that the removing app/console is enough, but this line ensures that nobody will accidentally add it back to the repository.